pkgs/misc/vim-plugins/default.nix

```diff
@@ -4,6 +4,7 @@
 , xkb_switch, rustracerd, fzf, skim
 , python3, boost, icu, ncurses
 , ycmd, makeWrapper, rake
+, lua51Packages, ncurses, zlib, llvm_39
```

Comment:
Looks like meanwhile `ncurses` was already added to the inputs for something else.